.env.example

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY_BASE=
+BASE_URL=
+DATABASE_USER=
+DATABASE_PASSWORD=
+POSTGRES_MULTIPLE_DATABASES=project_tides_db,project_tides_test_db
+DATABASE_DB=project_tides_db
```

Comment:
let's do
```suggestion
DATABASE_DB=project_tides
```

Reply:
how is this different from `POSTGRES_MULTIPLE_DATABASES`?

Reply:
`POSTGRES_MULTIPLE_DATABASES` can have comma separated multiple database names that may include testing database. This will be difficult every time we will be using the main database name. So it is been used for main database name.

Reply:
We are using `POSTGRES_MULTIPLE_DATABASES` just for creating new databases while creating a new container.